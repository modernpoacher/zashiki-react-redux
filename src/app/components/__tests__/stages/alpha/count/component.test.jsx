import React from 'react'

import {
  toSnapshot
} from 'react-component-snapshot'

import '@testing-library/jest-dom'

import {
  render
} from '@testing-library/react'

import Count from '#zashiki-react-redux/app/components/stages/alpha/count/component'

describe('#zashiki-react-redux/app/components/stages/alpha/count/component', () => {
  describe('Always', () => {
    it('renders', () => {
      expect(toSnapshot(render(
        <Count
          index={1}
          count={2}
        />
      )))
        .toMatchSnapshot()
    })
  })
})
