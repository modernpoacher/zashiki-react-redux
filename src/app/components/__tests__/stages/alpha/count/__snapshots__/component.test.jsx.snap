// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#zashiki-react-redux/app/components/stages/alpha/count/component Always renders 1`] = `
<h3>
  2
   of 
  2
</h3>
`;
